.github/workflows/scripts/ti_build/entry.py

```diff
@@ -53,11 +53,20 @@ def build_wheel(python: Command, pip: Command) -> None:
     elif wheel_tag:
         proj_tags.extend(["egg_info", f"--tag-build={wheel_tag}"])
 
-    if platform.system() == "Linux":
+    u = platform.uname()
+    if (u.system, u.machine) == ("Linux", "x86_64"):
         if is_manylinux2014():
             extra.extend(["-p", "manylinux2014_x86_64"])
         else:
             extra.extend(["-p", "manylinux_2_27_x86_64"])
+    elif (u.system, u.machine) in (("Linux", "arm64"), ("Linux", "aarch64")):
+        if is_manylinux2014():
```

Comment:
Note to self: we should probalby get rid of manylinux2014, since we no longer use it. But fine with leaving it in for now.

Reply:
i know, manylinux_2014 is deprecated